server/src/main/java/com/scalar/db/server/AdminService.java

```diff
@@ -33,8 +33,13 @@ public void pause(PauseRequest request, StreamObserver<Empty> responseObserver)
     if (request.getWaitOutstanding()) {
       LOGGER.warn("Pausing... waiting until outstanding requests are all finished");
       boolean drained = false;
+      long maxPauseWaitTime =
+          request.getMaxPauseWaitTime() != 0
```

Comment:
if `max_pause_wait_time` is not set, getMaxPauseWaitTime will return `0`.

reference:
https://developers.google.com/protocol-buffers/docs/javatutorial

```
a system default is used: zero for numeric types, the empty string for strings, false for bools. 
```